lib/typing/ext/parse.go

```diff
@@ -24,10 +25,25 @@ func ParseFromInterface(val any, additionalDateFormats []string) (*ExtendedTime,
 	return extendedTime, nil
 }
 
-// ParseTimeExactMatch is a wrapper around time.Parse() and will return an extra boolean to indicate if it was an exact match or not.
+// ParseTimeExactMatch - This function is the same as `ParseTimeExactMatchLegacy` with the only exception that it'll return if it was not an exact match
```

Comment:
"it'll return if it was not an exact match" - should this say "it'll return an error if it was not an exact match"?